src/portal/src/i18n/lang/zh-tw-lang.json

```diff
@@ -1851,36 +1857,36 @@
     "DATE_PICKER_SELECTED_LABEL": "{FULL_DATE} - 已選擇"
   },
   "BANNER_MESSAGE": {
-    "BANNER_MESSAGE": "Banner Message",
-    "MESSAGE_TYPE": "Message type",
-    "CLOSABLE": "Closable",
-    "FROM": "From",
-    "TO": "To",
-    "SUCCESS": "Success",
-    "INFO": "Info",
-    "WARNING": "Warning",
-    "DANGER": "Danger",
-    "ENTER_MESSAGE": "Enter your message here"
+    "BANNER_MESSAGE": "橫幅訊息",
+    "MESSAGE_TYPE": "訊息類型",
+    "CLOSABLE": "可關閉",
+    "FROM": "從",
+    "TO": "至",
+    "SUCCESS": "成功",
+    "INFO": "資訊",
+    "WARNING": "警告",
+    "DANGER": "危險",
+    "ENTER_MESSAGE": "在此輸入您的訊息"
   },
   "SECURITY_HUB": {
-    "SECURITY_HUB": "Security Hub",
-    "ARTIFACTS": "artifact(s)",
-    "SCANNED": "scanned",
-    "NOT_SCANNED": "not scanned",
-    "TOTAL_VUL": "Total Vulnerabilities",
-    "TOTAL_AND_FIXABLE": "{{totalNum}} total with {{fixableNum}} fixable",
-    "TOP_5_ARTIFACT": "Top 5 Most Dangerous Artifacts",
-    "TOP_5_CVE": "Top 5 Most Dangerous CVEs",
+    "SECURITY_HUB": "安全中心",
+    "ARTIFACTS": "Artifact(s)",
```

Comment:
```suggestion
    "ARTIFACTS": "產出物",
```

Reply:
I just looked at examples from other companies, and it looks like no one is translating this word. I think we can keep Artifacts going until there are tons of examples.